File: tuan01/tong.jl
function main()
    tokens = Int64[]
    for line in readlines(stdin)
        for token in split(line)
            push!(tokens, parse(Int64, token))
        end
    end
    if isempty(tokens) return end
    n = tokens[1]
    arr = tokens[2:n+1]
    println("$(sum(arr)) $(maximum(arr))")
end

main()

File: tuan01/tong.py
import sys

def main():
    input_data = sys.stdin.read().split()
    if not input_data:
        return
    n = int(input_data[0])
    arr = [int(x) for x in input_data[1:n+1]]
    print(f"{sum(arr)} {max(arr)}")

if __name__ == "__main__":
    main()